src/toolchain/export-android.md

```diff
@@ -0,0 +1,105 @@
+-->
+
+# Export to Android
+
+NOTE: Keep track of where you install the Android Command Line Tools and JDK 17
+
+Exporting with gdext for Godot requires some of the same pieces that are required for building Godot from source.
+Specifically, the Android SDK Command Line Tools and JDK 17 as mentioned in Godot's documentation
+[here](https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#requirements)
+
+Once you have those installed, you then need to follow Godot's instructions for setting up the build system
+[here](https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#setting-up-the-buildsystem)
+
+As of writing this, it is not entirely clear if JDK 17 is required in order for Android Command Line Tools to function properly.
+It has not been not tested without it. To be safe, it is included in the documentation for now.
+
+
+## Compiling
+
+The environment variable `CLANG_PATH` is used by bindgen's clang-sys dependency. See also
+[clang-sys documentation](https://github.com/KyleMayes/clang-sys?tab=readme-ov-file#environment-variables)
+
+Set the environment variable `CLANG_PATH` to point to Android's build of clang. Example:
+
+```bash
+export CLANG_PATH = "{androidCliDirectory}/{androidCliVersion}/ndk/{ndkVersion}/toolchains/llvm/prebuilt/{hostMachineOs}/bin/clang"
+```
+
+Then set the `CARGO_TARGET_{targetTripleHere}_LINKER` to point to the Android linker for the Android triple you are targeting.
+You need to compile your gdext library for each Android triple individually. Possible targets can be found by running:
+
+```bash
+rustup target list
+```
+
+You can find the linkers in the Android CLI directory at:
+
+```text
+"{androidCliDirectory}/{androidCliVersion}/ndk/{ndkVersion}/toolchains/llvm/prebuilt/{hostMachineOs}/bin/{targetTripleHere}{androidVersion}"
+```
+
+As of writing this, the tested tuples are:
+
+| Tuple                       | Environment Variable                            | Godot Arch     | GDExtension Config            |
+| --------------------------- | ----------------------------------------------- | -------------- | ----------------------------- |
+| `aarch64-linux-android`     | `CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER`     | `arm64`        | `android.debug.arm64`         |
+| `x86_64-linux-android`      | `CARGO_TARGET_X86_64_LINUX_ANDROID_LINKER`      | `x86_64`       | `android.debug.x86_64`        |
+| `armv7-linux-androideabi`   | `CARGO_TARGET_ARMV7_LINUX_ANDROID_LINKER`       | `arm32`        | `android.debug.armeabi-v7a`   |
+| `i686-linux-android`        | `CARGO_TARGET_I686_LINUX_ANDROID_LINKER`        | `x86_32`       | `android.debug.x86`           |
+
+Notice how the environment variables are in all caps and the triple's "-" is replaced with "_"
+
+Make sure to add all of the triples you want to support to the ``rustup`` via:
+
+```bash
+rustup target add {androidTripleHere}
+```
```

Comment:
Make sure you name variables consistently throughout the page, otherwise it's confusing.

This one was referenced as `{targetTriple}`, no? (as mentioned, omit the "here" part).